types/session_server.pp

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
can you document the new type with puppet-strings?
```suggestion
# @summary what the type does
```

please also add a unit test for the type

Reply:
Done and done :)